src/BuildASub.java

```diff
@@ -5,9 +5,17 @@
     public static void main(String[] args) throws Exception {
         String bread = chooseBread();
         String meat = chooseMeat();
+        String cheese = ChooseCheese();
+        Boolean toast = DoYouWantToasted();
 
         System.out.println("Your chosen bread is: " + bread);
         System.out.println("Your chosen meat is: " + meat);
+        System.out.println("Your chosen cheese is: " + cheese);
```

Comment:
This bit isn't required for the task. There is a different feature for printing the contents